### DISCUSSION TITLE
Enable JavaScript and cookies to continue

### DISCUSSION THREAD

**Discussion by goodair220917** [danny-avila/LibreChat#239] [Troubleshooting]
When I chose chatGPTBrowser, and sent a message, it replied:
An error occurred. Please try again in a few moments. Error message: Failed to send message. HTTP 403 - <!DOCTYPE html> <html lang="en-US"> <head> <title>Just a moment...</title> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta http-equiv="X-UA-Compatible" content="IE=Edge"> <meta name="robots" content="noindex,nofollow"> <meta name="viewport" content="width=device-width,initial-scale=1"> <link href="/cdn-cgi/styles/challenges.css" rel="stylesheet"> </head> <body class="no-js"> <div class="main-wrapper" role="main"> <div class="main-content"> <noscript> <div id="challenge-error-title"> <div class="h2"> <span class="icon-wrapper"> <div class="heading-icon warning-icon"></div> </span> <span id="challenge-error-text"> Enable JavaScript and cookies to continue </span> </div> </div> </noscript> <div id="trk_jschal_js" style="display:none;background-image:url('/cdn-cgi/images/trace/managed/nojs/transparent.gif?ray=7c529e4c5f49050e')"></div> <form id="challenge-form" action="/api/conversation?__cf_chl_f_tk=Q7uI9vzjgZyeT13sseYCdTqV8WJV07Phb_saVTTeDdU-1683726281-0-gaNycGzNEHs" method="POST" enctype="application/x-www-form-urlencoded"> <input type="hidden" name="md" value="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"> </form> </div> </div> <script> (function(){ window._cf_chl_opt={ cvId: '2', cZone: 'ai.fakeopen.com', cType: 'managed', cNounce: '96736', cRay: 'XXXXXXXXXXX', cHash: 'XXXXXXXX', cUPMDTk: "\/api\/conversation?__cf_chl_tk=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", cFPWv: 'g', cTTimeMs: '1000', cMTimeMs: '0', cTplV: 5, cTplB: 'cf', cK: "", cRq: { ru: 'XXXXXXXXXXXXXXXXXXXXXXXXX', ra: 'XXXXXXXXXXXXXXXXXXXXXXX', rm: 'XXXXXXXXX', d: 'XXXXXXXXXXXXXXXXXXXXXXXXX', t: 'XXXXXXXXXXXXXXXXXX', m: 'XXXXXXXXXXXXXXXXXX', i1: 'XXXXXXXXXXXXXXX', i2: 'XXXXXXXXXXXXXXXXX', zh: 'XXXXXXXXXXXXXXXX', uh: 'XXXXXXXXXXXXXX', hh: 'XXXXXXXXXXXXXXXXXXXX', } }; var trkjs = document.createElement('img'); trkjs.setAttribute('src', '/cdn-cgi/images/trace/managed/js/transparent.gif?ray=7c529e4c5f49050e'); trkjs.setAttribute('alt', ''); trkjs.setAttribute('style', 'display: none'); document.body.appendChild(trkjs); var cpo = document.createElement('script'); cpo.src = '/cdn-cgi/challenge-platform/h/g/orchestrate/managed/v1?ray=7c529e4c5f49050e'; window._cf_chl_opt.cOgUHash = location.hash === '' && location.href.indexOf('#') !== -1 ? '#' : location.hash; window._cf_chl_opt.cOgUQuery = location.search === '' && location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search; if (window.history && window.history.replaceState) { var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash; history.replaceState(null, null, "\/api\/conversation?__cf_chl_rt_tk=XXXXXXXXXXXXXXXXXXXXX" + window._cf_chl_opt.cOgUHash); cpo.onload = function() { history.replaceState(null, null, ogU); }; } document.getElementsByTagName('head')[0].appendChild(cpo); }()); </script> </body> </html>

**Comment by danny-avila** [comment]
it works for me. have you set your token from https://chat.openai.com/api/auth/session?
